openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceMainTabContent.tsx

```diff
@@ -131,9 +149,96 @@ function ServiceMainTabContent({
     setIsEdit(false);
   };
 
+  const callServicePatchAPI = async (
+    serviceCategory: ServiceTypes,
+    id: string,
+    jsonPatch: any
+  ) => {
+    switch (serviceCategory) {
+      case 'databaseServices':
```

Comment:
We can use the `ServiceTypes` enum for the cases name instead of hardcoded one

Reply:
Used `ServiceTypes` enum instead of strings.